pkg/generator/ingress_translator.go

```diff
@@ -191,7 +191,7 @@ func (translator *IngressTranslator) translateIngress(ctx context.Context, ingre
 					// If the service is of type ExternalName, we add a single endpoint.
 					typ = v3.Cluster_LOGICAL_DNS
 					publicLbEndpoints = []*endpoint.LbEndpoint{
-						envoy.NewLBEndpoint(service.Spec.ExternalName, uint32(externalPort)),
+						envoy.NewLBEndpoint(service.Spec.ExternalName, uint32(externalPort)), //#nosec G115
```

Comment:
We can't avoid that https://discuss.kubernetes.io/t/why-are-int32-s-all-over-the-place-where-uint32-s-are-apparently-more-appropriate-number-of-port-etc/21882 :D